templates/mobile/activation/activation.1.schema.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "activation",
+  "properties": {
+    "identifier": {
+      "type": "string",
+      "description": "hashed version of Google ID"
+    },
+    "client_id": {
+      "type": "string",
+      @COMMON_PATTERN_UUID_1_JSON@,
+      "description": "telemetry client_id, included only if identifer is empty"
+    },
+    "manufacturer": {
+      "type": "string",
+      "description": "Build.MANUFACTURER"
+    },
+    "model": {
+      "type": "string",
+      "description": "Build.MODEL"
+    },
+    "device": {
+      "type": "string"
+    },
+    "distribution_id": {
```

Comment:
Note use of snake case here rather than camelCase `distributionId`. We have historical pings that use camelCase field names, but snake case is the modern standard and I don't think we have any technical reason to keep this consistent with the core ping schema.